package.json

```diff
@@ -18,39 +18,46 @@
     "url": "https://github.com/bitmovin/bitmovin-player-ui/issues"
   },
   "homepage": "https://github.com/bitmovin/bitmovin-player-ui#readme",
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "Firefox ESR"
+  ],
   "devDependencies": {
+    "@inrupt/jest-jsdom-polyfills": "^1.4.3",
     "@types/jest": "^24.0.11",
     "@types/jsdom": "^12.2.4",
-    "autoprefixer": "^6.5.2",
+    "autoprefixer": "^10.4.13",
     "bitmovin-player": "^8.28.0",
     "browser-sync": "^2.26.3",
     "browserify": "^13.1.1",
-    "cssnano": "^3.8.0",
+    "cssnano": "^5.1.14",
     "del": "^2.2.2",
     "gulp": "^4.0.0",
     "gulp-css-base64": "^1.3.4",
     "gulp-header": "^2.0.5",
-    "gulp-postcss": "^6.2.0",
+    "gulp-postcss": "^9.0.1",
     "gulp-rename": "^1.4.0",
     "gulp-replace": "^0.6.1",
     "gulp-sass": "^5.1.0",
     "gulp-sass-lint": "^1.4.0",
     "gulp-sourcemaps": "^2.6.4",
     "gulp-tslint": "^8.1.3",
-    "gulp-typescript": "^3.2.4",
+    "gulp-typescript": "^5.0.1",
     "gulp-uglify": "^2.0.0",
-    "jest": "^24.7.1",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
     "merge2": "^1.2.2",
     "postcss-svg": "^1.0.6",
     "sass": "^1.49.8",
     "stream-combiner2": "^1.1.1",
-    "ts-jest": "^24.0.2",
+    "ts-jest": "^29.0.3",
     "tsify": "^2.0.2",
     "tslint": "^5.11.0",
-    "typescript": "^3.9.6",
+    "typescript": "4.3",
```

Comment:
nit: Please also use a proper semver here. It should be `^4.3.5`
(and run `npm install` to apply it to the `package-lock.json` file as well)

Reply:
Makes sense, done in ad046897.